services/app-web/src/components/SubmissionSummarySection/SubmissionTypeSummarySection/SubmissionTypeSummarySection.tsx

```diff
@@ -71,6 +81,18 @@ export const SubmissionTypeSummarySection = ({
                         data={SubmissionTypeRecord[submission.submissionType]}
                     />
                 </DoubleColumnGrid>
+                {showRateCertAssurance && (
+                    <DoubleColumnGrid>
+                        <DataDetail
+                            id="riskBasedContract"
+                            label="Is this a risk based contract"
+                            explainMissingData={!isSubmitted}
+                            data={booleanAsYesNoUserValue(
+                                submission.riskBasedContract
+                            )}
+                        />
+                    </DoubleColumnGrid>
+                )}
```

Comment:
The [figma](https://www.figma.com/file/a0PPSr65q1G798zV4bhzzO/%5BDeprecated%5D-State-submission-form?node-id=7297%3A55450&t=TMJz2jnML7OaQB0b-0) for the submission type summary orders the sections differently. Is that a part of another story? If not it could be done by putting all the sections in the same `DoubleColumnGrid`, expect for `Submission Description` which takes up an entire row.

I know `Contract action type` is part of another story, so at least without that the change in order would not need updated tests.

The thing to take note though, is the Figma linked above and in the Jira ticket is from our deprecated figma so we may need to touch base with designers if we wanted to make the change here.

Reply:
Good point! Wasn't thinking about placement yet because contract action type is not even on the summary yet. But it doesn't  hurt to wrap everything in the same DoubleColumnGrid - will do that.